src/libraries/System.IO.Hashing/src/System.IO.Hashing.csproj

```diff
@@ -4,6 +4,8 @@
     <Nullable>enable</Nullable>
     <TargetFrameworks>$(NetCoreAppCurrent);netstandard2.0;net461</TargetFrameworks>
     <IsPackable>true</IsPackable>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <ServicingVersion>1</ServicingVersion>
     <!-- TODO: Remove when the package ships with .NET 6. -->
     <DisablePackageBaselineValidation>true</DisablePackageBaselineValidation>
```

Comment:
You can remove this while you are here (and in main).  That will enable compat checking with the `6.0.0` package.

Reply:
Looks like something still wants it to check compat with 5.0.0:

```
src/libraries/System.IO.Hashing/src/System.IO.Hashing.csproj : error NU1102: Unable to find package System.IO.Hashing with version (= 5.0.0)
```

Reply:
In order for it to look for the 6.0.0 package on the main branch you'd need this change: https://github.com/dotnet/runtime/pull/61437

Reply:
@safern -- I think that means we should add https://github.com/dotnet/runtime/pull/61437 to the servicing branch.

Reply:
Jinx 😁 Maybe the way to handle this is to instead remove this property from System.IO.Hashing in #61437 and then bring that whole change to servicing?

Reply:
Main was refactored so we would have to do something a little different on servicing, but yeah, I can do something on release/6.0 so that it runs package validation against the 6.0.0 version of the package:

https://github.com/dotnet/runtime/blob/e9036b04357e8454439a0e6cf22186a0cb19e616/eng/packaging.targets#L13

Reply:
I included the change directly on this PR